packages/client/src/subscribe/resendStream.ts

```diff
@@ -13,10 +14,11 @@ const { ControlMessage } = ControlLayer
  * Sends resend request, handles responses.
  */
 
-export default function resendStream(client, opts = {}, onFinally = async () => {}) {
+export default function resendStream(client: StreamrClient, opts = {}, onFinally = async () => {}) {
     const options = validateOptions(opts)
     const { connection } = client
     const requestId = counterId(`${options.key}-resend`)
+    // @ts-expect-error
```

Comment:
It can be helpful to write after the @ts-expect-error directive why this is being used.

Reply:
you're right